docs/website/Makefile

```diff
@@ -10,4 +12,7 @@ all:
 	cp mtouch-errors.md $(WEBSITE_DOCS)/guides/ios/troubleshooting/mtouch-errors/index.md
 	cp binding_objc_libs.md $(WEBSITE_DOCS)/guides/ios/advanced_topics/binding_objective-c/binding_objc_libs/index.md
 	cp binding_types_reference_guide.md $(WEBSITE_DOCS)/guides/ios/advanced_topics/binding_objective-c/binding_types_reference_guide/index.md 
-	
+
+xamarin-analysis-doc:
```

Comment:
Documentation PR here: https://github.com/xamarin/documentation/pull/1819

I just added `XIA0001` to the `index.md` file because xamarin-macios/maccore:cycle8-xi point to a `xamarin-analysis` hash where there's only the first rule.